src/types/Event.ts

```diff
@@ -8,11 +8,9 @@ export type Event = {
   description: string;
   level: string;
   createdAt: string;
+  ownerId: string;
   // not in API
```

Comment:
That backend guy! :D

Reply:
every frontend girl needs some backend guy